todo_app/app.py

```diff
@@ -29,36 +57,30 @@ def index():
          list.append(Item(card["_id"], card["Name"], status))
       print (f"The record is {json_record_for_todos}" )
       view_model = ViewModel(list)
-      return  render_template('index.html', view_model=view_model)
-
-      
+      if app.config['LOGIN_DISABLED']:
+         user_role = "Writer"
+      else:
+          user_role = current_user.role
+      return  render_template('index.html', view_model=view_model, user_role = user_role)
 
    @app.route('/addNewTitle',methods = ['POST'])
    def addTitle():
-      todo = request.form.get('nm')
-      print(todo)
-      collection.insert_one({"Name": todo, "Status": "To Do" })
-      return redirect(url_for('index'))  
+      if (current_user.role == "Writer"):
```

Comment:
If you compare this to the function above, this _doesn't_ account for the possibility that the `LOGIN_DISABLED` flag is set. If that was set, this would error (since the `current_user` variable is undefined. 
The simplest solution would likely be to explicitly check:
```suggestion
    if (app.config['LOGIN_DISABLED'] or current_user.role == "Writer"):
```
But possibly nicer would be to extract this out into a function (since we'll want to reuse the same form the `updateCard` endpoint too)